src/Middleware/RequestThrottling/src/Internal/TailDrop.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.RequestThrottling.Internal
+{
+    internal class TailDrop : RequestQueue
```

Comment:
We shouldn't just change the default to TailDrop. Keep all the existing tests fro the base RequestQueue and add special tests for TailDrop.

Reply:
Hopefully will have multiple queue implementations soon, will run the existing tests on all of them.